doc/cabal-commands.rst

```diff
@@ -491,7 +491,7 @@ description file or freeze file.
 
 ``cabal outdated`` supports the following flags:
 
-.. option:: --v1-freeze-file
+.. option:: --freeze-file
```

Comment:
I'm not sure about this change. I think `cabal outdated` does work only with `v1`-things.

Reply:
This is what we have rendered:

```
ghci> :main outdated --help
Check for outdated dependencies.

Usage: <interactive> outdated [FLAGS] [PACKAGES]

Checks for outdated dependencies in the package description file or freeze
file

Flags for outdated:
 -h, --help                     Show this help text
 --project-dir=DIR              Set the path of the project directory
 --project-file=FILE            Set the path of the cabal.project file
                                (relative to the project directory when
                                relative)
 -v[n], --verbose[=n]           Control verbosity (n is 0--3, default
                                verbosity level is 1)
 --freeze-file                  Act on the freeze file
 --v2-freeze-file               Act on the new-style freeze file (default:
                                cabal.project.freeze)
 --simple-output                Only print names of outdated dependencies, one
                                per line
 --exit-code                    Exit with non-zero when there are outdated
                                dependencies
 -q, --quiet                    Don't print any output. Implies '--exit-code'
                                and '-v0'
 --ignore=PKGS                  Packages to ignore
 --minor[=PKGS]                 Ignore major version bumps for these packages
```